hack/test-end-to-end-docker.sh

```diff
@@ -36,7 +36,7 @@ function cleanup()
 	dump_container_logs
 
 	echo "[INFO] Dumping all resources to ${LOG_DIR}/export_all.json"
-	oc export all --all-namespaces --raw -o json --config=${ADMIN_KUBECONFIG} > ${LOG_DIR}/export_all.json
+	oc export all --all-namespaces --raw -o json --config=${ADMIN_KUBECONFIG-} > ${LOG_DIR}/export_all.json
```

Comment:
Alright. I was hoping that the detection there was going to be more involved than `set +e` and ignoring the failure, but alright. Comment about syntax remains.